Javascript/Housey pricey/house.js

```diff
@@ -0,0 +1,37 @@
+// Peter's parameters
+let peterWidth = 8;
+let peterHeight = 10;
+let peterDepth = 10;
+let peterGardenSize = 100;
+let peterRealPrice = 2500000;
+
+// Julia's parameters
+let juliaWidth = 5;
+let juliaHeigth = 8;
+let juliaDepth = 11;
+let juliaGardenSize = 70;
```

Comment:
All those variables could also be set through an object. Then you can access the properties as `peterHouse.depth`,  `peterHouse.width` for example. Nothing wrong with what you've written - this is just another suggestion ( :

Reply:
****Thank you. Next time i will try to write better code!****